pkg/util/loop/loop.go

```diff
@@ -154,8 +156,7 @@ func (loop *Device) shareLoop(imageIno, imageDev uint64, mode int, number *int)
 			// be sure that the loop device won't be released between this
 			// check and the mount of the filesystem
 			sylog.Debugf("Sharing loop device %d", device)
-			loop.fd = new(int)
-			*loop.fd = loopFd
+			loop.fd = &loopFd
```

Comment:
@cclerget loop.fd is not thread-safe, and not protected here.

Reply:
`loop` is a short-live object not a singleton object and `AttachFromFile` is not intended to be called in parallel for the same instance